src/controllers/mintController.ts

```
@@ -263,7 +262,20 @@ const fintechTxIdSchema = z
   .optional();
 
 export const depositBodySchema = z.object({
-  currency: z.string().length(3).toUpperCase(),
+  currency: z
+    .string()
+    .length(3)
+    .transform((value) => value.toUpperCase())
+    .refine(
+      (currency) =>
+        isAllowedDepositCurrency(currency) || isForbiddenDepositCurrency(currency),
```

ESLint

[error] Delete `⏎·······`

(prettier/prettier)